src/ApparentHorizons/Tags.hpp

```diff
@@ -361,6 +361,17 @@ struct AreaCompute : Area, db::ComputeTag {
   using argument_tags =
       tmpl::list<StrahlkorperTags::Strahlkorper<Frame>, AreaElement<Frame>>;
 };
+/// The Irreducible (areal) mass of an apparent horizon
+struct IrreducibleMass : db::SimpleTag {
+  using type = double;
+};
+/// Computes the Irreducible mass of an apparent horizon from its area
+template <typename Frame>
+struct IrreducibleMassCompute : IrreducibleMass, db::ComputeTag {
+  using base = IrreducibleMass;
+  static constexpr auto function = ::StrahlkorperGr::irreducible_mass;
```

Comment:
This needs to be changed to a mutating compute item now. E.g.
```cpp
static void function(const gsl::not_null<double*> result, const double area) noexcept {
  *result = ::StrahlkorperGr::irreducible_mass(area);
}
```
Since this is dealing with `double`s I'm not sure we need a mutating overload of the `irreducible_mass` function

Reply:
Could you please clarify why this change is needed? Also, are you saying we can observe mirr without this tag? If so, how?

Reply:
The `function` for `db::ComputeTag`s should return void and take a `gsl::not_null<>` to mutate the result.  We have been converting all the compute tags to do this as part of simplifying DataBox and (though not important in this case) reducing memory allocations.  In most cases the `function` should call a mutating overload of a free function, but for fundamental types it is not necessary.    Also, you still need this tag in order to observe...

Reply:
Checks complete